fuzz/fuzz_targets/transactions_derivation.rs

```diff
@@ -0,0 +1,72 @@
+#![no_main]
+use libfuzzer_sys::fuzz_target;
+
+use libfuzzer_sys::arbitrary::Arbitrary;
+
+use revault_tx::{
+    miniscript::bitcoin::{
+        blockdata::constants::max_money, hashes::Hash, secp256k1::SECP256K1, Network, OutPoint,
+        Txid,
+    },
+    transactions::tests_helpers::{derive_transactions, seed_rng},
+};
+
+#[derive(Arbitrary, Debug)]
+struct Config {
+    n_stk: usize,
+    n_man: usize,
+    csv: u32,
+    deposit_txid: [u8; 32],
+    deposit_vout: u32,
+    deposit_value: u64,
+    feebump_txid: [u8; 32],
+    feebump_vout: u32,
+    feebump_value: u64,
+    unvault_spends: Vec<([u8; 32], u32, u64)>,
+}
+
+fuzz_target!(|config: Config| {
+    if config.n_stk > 150 || config.n_stk < 2 || config.n_man > 150 {
```

Comment:
Wait, it works with N = M = 150? Wtf? In revaultd functional tests mscompiler would fail even with N = 40, M = 20. I don't get it... 

```
diff --git a/tests/test_misc.py b/tests/test_misc.py
index 816aecf..0dc6dbf 100644
--- a/tests/test_misc.py
+++ b/tests/test_misc.py
@@ -130,7 +130,7 @@ def test_largewallets(revaultd_stakeholder, bitcoind):
 @pytest.mark.skipif(not POSTGRES_IS_SETUP, reason="Needs Postgres for servers db")
 def test_getdepositaddress(revault_network, bitcoind):
     rn = revault_network
-    rn.deploy(4, 2)
+    rn.deploy(40, 20)
     stk = rn.stk(0)
     addr = stk.rpc.call("getdepositaddress")["address"]
    ```

Reply:
No, it was a higher bound. `150` is ridiculously small for an u32 already, and i wanted to have it hit some errors.